### ISSUE TITLE
Scraping any account, including the demo account in the video in the readme, fails.

### ISSUE THREAD

**Issue by kristopolous** [arc298/instagram-scraper#690]
This is what happens:

```
$ instagram-scraper demo_ig_scraper    
ERROR: Error getting user details for demo_ig_scraper. Please verify that the user exists.
```

**Comment by arc298** [comment]
demo_ig_scraper was a dummy user for demonstrative purposes and no longer exists. You can verify yourself if the account exists and is accessible by visiting `https://www.instagram.com/<username>`

**Comment by lerockit** [comment]
I'm facing the same issue for all Instagram users. Here's an example:

![image](https://user-images.githubusercontent.com/33522617/124969747-76560180-dffd-11eb-8215-a1fc4c181526.png)

https://www.instagram.com/zuck/

**Comment by oukAyou** [comment]
I get this error every time I try to scrape a user's data : 

"ERROR: Error getting user details for USERNAME. Please verify that the user exists."

The user exists when I try to access to his profile via browser.
I think Instagram have found a way to stop finding profiles using this library. Any help please ??

**Comment by kristopolous** [comment]
Right, it's broken for all as in every account. Sorry if that wasn't clear

**Comment by oukAyou** [comment]
Hello everybody, try to **upgrade to 1.10.0 version**, I stopped having this problem after upgrade. Good luck :)

**Comment by servin** [comment]
I still have the same problem, created a new environment for my RaspberryPi and only ran once, same error after that

**Comment by AlexNik** [comment]
@servin try to scrape whith login/password option.
